src/Builder/FormContractor.php

```diff
@@ -134,8 +127,7 @@ public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescrip
                     ));
                 }
             }
-            // NEXT_MAJOR: Check only against FQCNs when dropping support for Symfony 2.8
-        } elseif ('sonata_type_admin' === $type || $this->checkFormClass($type, [AdminType::class])) {
```

Comment:
This could cause problems. We dropped a named form type. You can still define a named form type in symfony 5 @greg0ire

Reply:
Only the symfony (internal) form types removed this feature. This is not related to any other 3rd party form

Reply:
Then maybe we should revert and change the comments